tests/test_hardhat.py

```diff
@@ -94,19 +94,19 @@ def test_set_block_gas_limit(hardhat_provider):
     assert hardhat_provider.set_block_gas_limit(gas_limit) is True
 
 
-def test_sleep(hardhat_provider):
+def test_set_timestamp(hardhat_provider):
     seconds = 5
-    time_1 = hardhat_provider.sleep(seconds)
-    time_2 = hardhat_provider.sleep(seconds)
+    pending_time_stamp = hardhat_provider.get_block("pending").timestamp
+    time_1 = hardhat_provider.set_timestamp(seconds + pending_time_stamp)
```

Comment:
`set_timestamp()` does not return anything, so we need to grab the pending timestamp again here